.github/workflows/pull-request.yml

```diff
@@ -28,6 +28,12 @@ jobs:
       - name: Run Linter
         run: |
           swiftlint --strict
+          
+      - name: Validate Pull Request Name
+        id: versioning
+        uses: Oliver-Binns/Versioning@main
```

Comment:
@Oliver-Binns should we be pinning to main or a specific commit SHA? I _think_ this goes against the GDS way but I can't find the right docs. 

There's a chance the maintainer of this package deploys a breaking change/defect